model/post.go

```diff
@@ -122,6 +122,12 @@ type PostForExport struct {
 	ReplyCount  int
 }
 
+type DirectPostForExport struct {
+	Post
+	User           string
+	ChannelMembers *[]string
```

Comment:
Making `ChannelMembers` a pointer here seems to be redundant (arrays are already pointers). Can we simplify?

Reply:
I'll change it since this is the same case of the arrays of `Users`.